### PR TITLE
[WIP] Feature: drag-and-drop files in specific folder

### PR DESCRIPTION
**Pull request by pvilalta**
https://platform-dev-specific-folder.bimdata.io

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] My code follows the code style of this project.
- [x] I have tested my code.

### CODE REVIEW

File: src/components/specific/groups/group-card/group-card-delete-guard/GroupCardDeleteGuard.vue [hunk 1]
```diff
@@ -14,7 +14,7 @@
     </div>
     <div class="group-card-delete-guard__message">
       <i18n-t keypath="GroupCardDeleteGuard.message" tag="span">
-        <template v-slot:name>
+        <template #name>
```
Comment (Kurtil) [inline src/components/specific/groups/group-card/group-card-delete-guard/GroupCardDeleteGuard.vue:17]: :P

File: src/services/FileService.js [hunk 2]
```diff
@@ -244,20 +244,18 @@ class FileService {
...
-
         const tree = createFolderTree(rootFolder, removeRootFolder(paths));
 
+        let rootFolderNode = null;
         function getRootFolderNode(node) {
           if (node.id === rootFolder.id) {
-            return node;
-          } else {
-            return node.children?.find(getRootFolderNode);
+            return (rootFolderNode = node);
```
Comment (Kurtil) [inline src/services/FileService.js]: return an affectation ?

File: src/services/FileService.js [hunk 3]
```diff
@@ -244,20 +244,18 @@ class FileService {
...
           }
+          return node.children?.forEach(getRootFolderNode);
```
Comment (Kurtil) [inline src/services/FileService.js]: forEach returns `undefined`.

**Review by Kurtil — CHANGES_REQUESTED** [review]

**Review by Kurtil — APPROVED** [review]

**Review by NicolasRichel — APPROVED** [review]

**Comment by NicolasRichel** [comment]
Replaced by #361

**Pull request closed without merging**